src/Service.Host/client/src/components/Root.js

```diff
@@ -71,6 +73,16 @@ const Root = ({ store }) => (
                                     path="/purchasing/reports/orders-by-supplier/:id"
                                     component={OrdersBySupplier}
                                 />
+                                <Route
+                                    exact
+                                    path="/purchasing/reports/orders-by-part"
+                                    component={OrdersByPartOptions}
+                                />
+                                <Route
+                                    exact
+                                    path="/purchasing/reports/orders-by-part/:id"
```

Comment:
How come you need the id in the path here? Is it a part number string? If so it could have forward slashes in it and cause some awkwardness

Reply:
You know I wondered that as I typed/copied it but wasn't awake enough to fully think it through, it should be /report I'm sure. Will change!